development/Dockerfile

```diff
@@ -13,8 +13,11 @@ RUN pip install --upgrade pip\
 # -------------------------------------------------------------------------------------
 # Install NetBox
 # -------------------------------------------------------------------------------------
+# Remove redis==3.4.1 from the requirements.txt file as a workaround to #4910
+# https://github.com/netbox-community/netbox/issues/4910, required for version 2.8.8 and earlier
```

Comment:
I'd rather just change `.travis.yml` to test with 2.8.9 instead of 2.8.3, personally.

Reply:
@carbonarok This change should not be a part of this PR